generator/altitude_generator.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "geometry/point2d.hpp"
+
+#include "indexer/feature_altitude.hpp"
+
+#include "std/string.hpp"
+
+namespace routing
+{
+class IAltitudeGetter
```

Comment:
Why? IAltitudeGetter is an interface with one pure virtual method. Don't we use prefix "I" for such classes?